addon/model/util/model-node-utils.ts

```diff
@@ -0,0 +1,30 @@
+import MapUtils from "@lblod/ember-rdfa-editor/model/util/map-utils";
+import ModelNode from "@lblod/ember-rdfa-editor/model/model-node";
+import ModelElement from "@lblod/ember-rdfa-editor/model/model-element";
+import {LIST_CONTAINERS} from "@lblod/ember-rdfa-editor/model/util/constants";
+
+export default class ModelNodeUtils {
+  static DEFAULT_IGNORED_ATTRS: Set<string> = new Set(["__dummy_test_attr", "__id"]);
```

Comment:
probably best to add `data-editor-position-level` and `data-editor-rdfa-position-level` to this list.

Both are used to make it possible to style the current `active` subtree based on your position, e.g. if you're cursor is inside an element it will have `data-editor-position-level`=1, it's parent will have 2 etc. but should not block two nodes from merging